### PR TITLE
Add info on use of CPNS deaths vs ONS deaths

### PR DESCRIPTION
**Pull request by LindaNab**

### CODE REVIEW

**Comment by iaindillingham** [comment]
I'm in two minds whether this information should be in `study-def-variables.md` or `dataset-cpns.md`. Either way, it should be in [an _admonition_, or call-out](https://squidfunk.github.io/mkdocs-material/reference/admonitions/). You can do that, like this:

```
For more information, see the [CPNS data section](dataset-cpns.md).

!!! note
    Please note that CPNS is restricted to in-hospital covid-related deaths, for covid-related deaths in any setting, ONS-registered deaths where cause of death matches [COVID-19 coding in ICD-10](https://www.who.int/classifications/icd/COVID-19-coding-icd10.pdf) is generally more useful.
```

**Comment by LindaNab** [comment]
Thanks. Changed to an admonition now!

File: docs/study-def-variables.md [hunk 1]
```diff
@@ -109,7 +109,10 @@ For more information, see the [SGSS data section](dataset-sgsscovid.md).
 ## CPNS
 
 These variables are derived from the COVID-19 Patient Notification System (CPNS), which collects info on all in-hospital covid-related deaths.
-For more information, see the [CPNS data section](dataset-cpns.md).
+For more information, see the [CPNS data section](dataset-cpns.md). 
+
+!!! note
+    Please note that CPNS is restricted to in-hospital covid-related deaths, for covid-related deaths in any setting, ONS-registered deaths where cause of death       matches [COVID-19 coding in ICD-10](https://www.who.int/classifications/icd/COVID-19-coding-icd10.pdf) is generally more useful.
```
Comment (StevenMaude) [inline docs/study-def-variables.md:115]: nit:
```suggestion
    CPNS is restricted to in-hospital covid-related deaths only. For covid-related deaths in any setting, ONS-registered deaths where cause of death matches [COVID-19 coding in ICD-10](https://www.who.int/classifications/icd/COVID-19-coding-icd10.pdf) is generally more useful.
```

Perhaps very slightly more readable :arrow_up: ?

Comment (iaindillingham) [reply docs/study-def-variables.md:115]: I'd agree 🙂 I like the "Commit suggestion" button, too.

**Review by iaindillingham — APPROVED** [review]
Looks good! Thanks for making those changes.

**Pull request merged**